File: PythonAbril30dias/dia9.py
days = int(input("Introduce los días: "))
for i in range(1, days+1):
    if i%3 == 0 and i%5 == 0:
        print("FP + Máster")
    elif i%3 == 0:
        print("FP")
    elif i%5 == 0:
        print("Máster")
    else:
        print(i)

File: JavaAbril30dias/src/Dia9.java
import java.util.Scanner;

public class Dia9 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.println("Introduce los días:");
        int days = input.nextInt();
        for (int i=1; i<days+1; i++) {
            if (i%3 == 0 && i%5 == 0) {
                System.out.println("FP + Máster");
            }
            else if (i%3 == 0) {
                System.out.println("FP");
            }
            else if (i%5 == 0) {
                System.out.println("Máster");
            }
            else {
                System.out.println(i);
            }
        }
    }
}
